tests/xoto3/dynamodb/put_test.py

```diff
@@ -0,0 +1,39 @@
+import os
+import random
+
+import pytest
+import boto3
+
+import xoto3.dynamodb.put as xput
+
+from tests.xoto3.dynamodb.testing_utils import make_pytest_put_fixture_for_table
+
+
+XOTO3_INTEGRATION_TEST_ID_TABLE_NAME = os.environ.get(
+    "XOTO3_INTEGRATION_TEST_DYNAMODB_ID_TABLE_NAME"
+)
+
+
+_INTEGRATION_ID_TABLE = (
+    boto3.resource("dynamodb").Table(XOTO3_INTEGRATION_TEST_ID_TABLE_NAME)
+    if XOTO3_INTEGRATION_TEST_ID_TABLE_NAME
+    else None
+)
+integration_table_put = make_pytest_put_fixture_for_table(_INTEGRATION_ID_TABLE)
```

Comment:
I do like the fixture approach better. but I also can't figure out how to make this work nicely with my dynamic fixture generator, which i'm not yet prepared to abandon.

The next time I have some time to try to make that fit the pytest pattern better (or have an example to work off of), I'll try to prefer this approach of having the fixture mark the test as being skipped, which definitely is cleaner and nicer.

And yeah, I've done the pytest option thing before - see vnxpy/xoipy's `runslow`. But more and more I find myself a fan of env vars rather than options, because it allows the same script to transparently accomplish different behavior. Env vars are composable in a way command line arguments aren't as easy to compose.